packages/templates/src/getMetaData.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import { TEMPLATE_NAMES } from './constants';
+
+export function getTemplateMetaData() {
+  return Promise.all(
+    TEMPLATE_NAMES.map((title) =>
+      import(
+        /* webpackChunkName: "chunk-web-stories-template-[index]-metaData" */ `./raw/${title}/metaData`
+      ).then((data) => [title, data.default])
```

Comment:
Since the metadata object already contains the `slug`, there's no need for this array here. We can just return the metadata object as-is.

Reply:
heads up! If you're gonna use the slug in this branch there are 4 slugs that need updating 🤕  

- 'celebrity-q-a' -> 'celebrity-q-and-a'
- 'food-stuff' -> 'food-and-stuff'
- 'fresh-bright' -> 'fresh-and-bright'
- 'vintage-chair-buying-guide' -> 'vintage-chairs-buying-guide'

Didn't catch these until i started chopping up the templates view, was going to make the fix there but you need it here now too.